src/Classes/ServiceAPI/MyRadio_Swagger2.php

```diff
@@ -348,6 +370,44 @@ private static function getTags($apis)
         return $tags;
     }
 
+    private static function getMethodOpType($method)
+    {
+        $name = $method->getName();
+
+        //Note the ordering is important - create is static!
+        if (
+            CoreUtils::startsWith($name, 'create') ||
+            CoreUtils::startsWith($name, 'add')
+        ) {
+            return 'post';
+        }
+
+        if (
+            $name === 'toDataSource' ||
+            CoreUtils::startsWith($name, 'get') ||
+            CoreUtils::startsWith($name, 'is') ||
+            $method->isStatic()
+        ) {
+            return 'get';
+        }
+
+        return 'put';
+    }
+
+    private static function getMethodPublicName($method) {
+        $name = $method->getName();
+
+        if ($name === 'toDataSource' || $name === 'create') {
```

Comment:
add findByOptions here instead, if it's never meant to be exposed? Or alternatively, add `@swagger ignore` directives to toDataSource & create functions